api/actions/createWorkItem.js

```diff
@@ -93,10 +93,10 @@ module.exports = {
                             'NEW',
                             foundInBuild,
                             priority,
-                            url,
+                            html_url,
                             recordTypeId);
                         const syncedItemFromGus = await Gus.getById(syncedItem.id);
-                        const displayUrl = (hideWorkItemUrl === true) ? syncedItemFromGus.Name : `[${syncedItemFromGus.Name}](${process.env.WORK_ITEM_BASE_URL + syncedItem.id}/view)`;
+                        const displayUrl = (hideWorkItemUrl === 'true') ? syncedItemFromGus.Name : `[${syncedItemFromGus.Name}](${process.env.WORK_ITEM_BASE_URL + syncedItem.id}/view)`;
```

Comment:
This was a bug that was preventing masking URL even when hideWorkItemUrl was set to True.